homeassistant/components/zwave_js/light.py

```diff
@@ -0,0 +1,302 @@
+            return
+
+        # TODO: setting duration needs to be fixed upstream
+        # https://github.com/zwave-js/node-zwave-js/issues/1321
+        return
+
+        if duration is None:  # type: ignore
+            # no transition specified by user, use defaults
+            duration = 7621  # anything over 7620 uses the factory default
+        else:
+            # transition specified by user
+            transition = duration
+            if transition <= 127:
+                duration = int(transition)
+            else:
+                minutes = int(transition / 60)
+                LOGGER.debug(
+                    "Transition rounded to %d minutes for %s",
+                    minutes,
+                    self.entity_id,
+                )
+                duration = minutes + 128
+
+        # only send value if it differs from current
+        # this prevents sending a command for nothing
+        if self._dimming_duration.value != duration:
+            await self.info.node.async_set_value(self._dimming_duration, duration)
+
+    @callback
+    def _calculate_color_values(self) -> None:
+        """Calculate light colors."""
+
+        # RGB support
+        red_val = self.get_zwave_value(
+            "currentColor", CommandClass.SWITCH_COLOR, property_key_name="Red"
+        )
+        green_val = self.get_zwave_value(
+            "currentColor", CommandClass.SWITCH_COLOR, property_key_name="Green"
+        )
+        blue_val = self.get_zwave_value(
+            "currentColor", CommandClass.SWITCH_COLOR, property_key_name="Blue"
+        )
+        if red_val and green_val and blue_val:
+            self._supports_color = True
+            # convert to HS
+            self._hs = color_util.color_RGB_to_hs(
+                red_val.value, green_val.value, blue_val.value
+            )
+
+        # Update color temp limits.
+        min_kelvin_val = self.get_zwave_value(81, CommandClass.CONFIGURATION)
+        if min_kelvin_val:
+            self._max_mireds = round(
+                color_util.color_temperature_kelvin_to_mired(min_kelvin_val.value)
+            )
+        max_kelvin_val = self.get_zwave_value(82, CommandClass.CONFIGURATION)
+        if max_kelvin_val:
+            self._min_mireds = round(
+                color_util.color_temperature_kelvin_to_mired(max_kelvin_val.value)
+            )
+
+        # White colors
+        ww_val = self.get_zwave_value(
+            "currentColor", CommandClass.SWITCH_COLOR, property_key_name="Warm White"
+        )
+        cw_val = self.get_zwave_value(
+            "currentColor", CommandClass.SWITCH_COLOR, property_key_name="Cold White"
+        )
+        if ww_val and cw_val:
+            # Color temperature (CW + WW) Support
+            self._supports_color_temp = True
+            # Calculate color temps based on whites
+            cold_level = cw_val.value or 0
+            if cold_level or ww_val.value:
+                self._color_temp = int(
```

Comment:
I used `round` in the other places.
```suggestion
                self._color_temp = round(
```